src/components/Home.js

```diff
@@ -1,26 +1,37 @@
 import React from 'react';
 import Beer from './Beer';
 
-class Home extends React.Component {
+export default ({
+  beers,
+  toggleLiked,
+  likedBeers,
+}) => {
+  const getLiked = beerId => likedBeers.find(({ id }) => id === beerId);
+
+  const renderBeers = () => beers.map(beer => {
+    const { id  } = beer;
 
-  render() {
     return (
-      <div className="Home">
-        <div className="site-title">
-          BrewDog Back Catalogue
-        </div>
-        <div className="subheading">
-          Explore here
-        </div>
-        <div className="beers-container">
-          {
-            this.props.beers.map(beer => <Beer key={beer.id} beer={beer} checkLiked={this.props.checkLiked} />)
```

Comment:
This is another thing that is largely a matter of opinion, but I don't like mapping directly in the jsx. As a personal rule, I abstract any logic from jsx if it is a mapping, or any more than one conditional:
✔️ :
```
<div>
 { isFavorited && <FavBanner /> }
</div>
```
❌ :
```
<div>
  { isFavorited ? <FavBanner /> : null }
  { isFavorited && !ignored && <FavBanner /> }
</div>
```